src/app/clients/ClientsPage.tsx

```diff
@@ -85,56 +84,70 @@ const ClientsPage: React.FC<{}> = () => {
     const [filteredClientCount, setFilteredClientCount] = useState<number>(0);
     const [sortState, setSortState] = useState<SortState<ClientsTableRow>>({ sortEnabled: false });
     const [primaryFilters, setPrimaryFilters] = useState<Filter<ClientsTableRow, any>[]>(filters);
+    const abortController = useRef<AbortController>();
```

Comment:
is it ok if I change to clientTableFetchAbortController, because the same abortController is used for both the count and the data requests (because if we abort one we will abort the other, and then we can just run abortController.abort() and it deals with both)

Reply:
that sounds reasonable!